build-support/bin/BUILD

```diff
@@ -36,6 +36,21 @@ python_binary(
   sources = ['generate_travis_yml.py'],
 )
 
+python_binary(
+  name = 'generate_docs',
+  sources = ['generate_docs.py'],
+  dependencies = [
+    '3rdparty/python:pystache',
+    '3rdparty/python:requests',
```

Comment:
These should be inferred :)

Reply:
Damn, I keep forgetting this is a thing now :)